lib/get_env.rb

```diff
@@ -1,9 +1,16 @@
 # frozen_string_literal: true
-require "get_env/version"
+
+require 'get_env/version'
+begin
+  require 'rubocop/cop/lint/no_env'
```

Comment:
I'm not a fan of always running this just in case it's loaded from rubocop.

Could we add a `lib/get_env/rubocop_cops.rb` that will be meant to be required from the `.rubocop.yml` file like this:

```yaml
require:
  ...
  - get_env/rubocop_cops
```